.github/workflows/tests.yml

```
@@ -1,52 +1,51 @@
-    - uses: dorny/paths-filter@v2
-      id: filter
-      with:
-        filters: |
-          codechange:
-            - '!presto-docs/**'
-
   test:
-    runs-on: ubuntu-latest
-    needs: changes
+    name: "presto-tests"
+    runs-on: &runner >-
```

actionlint (1.7.10)

[error] anchor "runner" is defined but not used

(syntax-check)